saleor/order/actions.py

```diff
@@ -1957,6 +1957,10 @@ def create_fulfillments_for_returned_products(
         ).delete()
 
         call_order_event(manager, WebhookEventAsyncType.ORDER_UPDATED, order)
+        if new_order:
+            call_order_event(
```

Comment:
Sugestion:
Maybe we could create `draft_order_created()` handler, and call it here, and in  `draftOrderCreate`. 
This could give us confirmation, that we have a single place of logic that should be called for created draft order.